apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -24,6 +24,7 @@ defmodule Aecore.Miner.Worker do
 
   @mersenne_prime 2147483647
   @coinbase_transaction_value 100
+  @max_block_size_bytes 500_000
```

Comment:
Please make max_block_size_bytes and Environment variable, so we can keep it separate for testing and "production"